### PR TITLE
Leadershipboard: Hit the API

### PR DESCRIPTION
**Pull request by wale-prog**
In this pull request, I have implemented the following in the project:
- I have generated the game Id with my chosen name.
- implemented the "refresh" button by using a function to receive data from the API and parsing the JSON.
- implemented the form "submit" button by using a function to send data to the API.
- I have used `async` and `await` JavaScript features to consume the API.

### CODE REVIEW

File: src/modules/displayUser.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+export function getUsers() {
+  const refreshBtn = document.querySelector('.refresh');
+  const listItems = document.querySelector('.list-items');
+  refreshBtn.addEventListener('click', async (e) => {
+    e.preventDefault();
+    listItems.innerHTML = '';
+    singleRun();
+  });
+}
```
Comment (youmari) [inline src/modules/displayUser.js]: - [ ] please try to use the `arrow function` instead of `function keyword` in order to practice working with both

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @wale-prog , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great title and short summary 
✔️ Good commit messages
✔️ No linter error

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @wale-prog,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by wale-prog** [comment]
Thanks for the feedback

**Pull request merged**